hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/storage/TestContainerCommandsEC.java

```diff
@@ -614,8 +614,15 @@ void testECReconstructionCoordinatorWith(List<Integer> missingIndexes)
     testECReconstructionCoordinator(missingIndexes, 3);
   }
 
+  @ParameterizedTest
+  @MethodSource("recoverableMissingIndexes")
+  void testECReconstructionCoordinatorWithPartialStripe(List<Integer> missingIndexes)
+      throws Exception {
+    testECReconstructionCoordinator(missingIndexes, 1);
+  }
+
   @Test
-  void testECReconstructionWithPartialStripe()
+  void testECReconstructParityWithPartialStripe()
           throws Exception {
     testECReconstructionCoordinator(ImmutableList.of(4, 5), 1);
   }
```

Comment:
nit: this can be removed, since `recoverableMissingIndexes` includes `(4, 5)`, so `testECReconstructionCoordinatorWithPartialStripe` covers the same case.

Reply:
Good point. I will fix this when I rebase #6515